src/mmw/js/src/core/utils.js

```diff
@@ -563,13 +563,30 @@ var utils = {
         return self.indexOf(value) === index;
     },
 
-    isInDrb: function(geom) {
-        var layers = settings.get('stream_layers'),
-            drb = _.find(layers, {code: 'drb_streams_v2'}).perimeter;
-
-        return !!intersect(geom, drb);
+    // Takes a geometry and returns an array of its coordinates
+    coordinates: function(geom) {
+        switch(geom.type) {
+            case 'MultiPolygon':
+                return _.flatten(_.flatten(geom.coordinates));
+            case 'Polygon':
+                return _.flatten(geom.coordinates);
+            default:
+                throw new Error('Unsupported geometry');
+        }
     },
 
+    // Check if a geom is completely within DRB_SIMPLE_PERIMETER
+    isInDrb: _.memoize(function(geom) {
```

Comment:
Memoized for efficiency.

`isInDrb` was previously only used to send Google Analytics, so this slight change in the behavior (of not counting partial intersections) should not break anything.